Formula/pdnsd.rb

```diff
@@ -24,6 +24,43 @@ def install
     system "make", "install"
   end
 
+  def caveats; <<-EOS.undent
+    Before starting "pdnsd" using "brew services" or manually, make sure that,
+    #{etc}/pdnsd.conf, #{var}/log/pdnsd.log and directory #{var}/cache/pdnsd
+    are all owned by "nobody".
```

Comment:
`owned by "nobody".` -> `owned by "nobody":` and indent the lines below by two spaces.